### DISCUSSION TITLE
Terminal Characteristics (QA)

### DISCUSSION THREAD

**Discussion by rchavez33-dev** [zed-industries/zed#7683] [Feature Requests]
I am starting to explore Zed, I am really liking it. However, I have a problem with the terminal, I use Oh-My-Posh, the problem is that the height of the line in the terminal makes it look weird and do not match the elements where they should go.

*Does anyone know what is the property that allows me to modify that or if it has not been added yet?

PS: I attach an image of what I am saying.
<img width="438" alt="Captura de pantalla 2024-02-11 a la(s) 23 05 35" src="https://github.com/zed-industries/zed/assets/66058913/4f1dc27c-2b22-40fb-ab08-a8c4de9a10b3">

**Comment by mikayla-maki** [comment]
The `line_height` setting, under `terminal`, can be set to `standard`, which should fix this problem:

```
// In your local settings file...
"terminal": {
  "line_height": "standard"
}
```

**Reply by rchavez33-dev** [reply]
Thank U!
